cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/EventBatchDurationCache.java

```diff
@@ -23,9 +23,12 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
 
 public class EventBatchDurationCache {
 
+    private static final Logger logger = LoggerFactory.getLogger(EventBatchDurationCache.class);
+
     public static class CacheKey {
```

Comment:
remove dependencies on CacheKey from outside class

Reply:
This class is for internal use and immutable, so to simplify it and make more readable you can remove getters